pkgs/applications/emulators/uxn/default.nix

```diff
@@ -6,13 +6,13 @@
 
 stdenv.mkDerivation rec {
```

Comment:
```suggestion
stdenv.mkDerivation {
```

Reply:
Ah I see why you remove pname now below. Why do you want to remove rec and the use of the pname variable?

Reply:
More recently, the use of `rec` is being avoided, because it triggers some undesirable behaviors like infinite recursions and bad interaction with overriding mechanisms.

In this specific case, just get rid of `pname` references around the code. It should suffice for now.